content/applications/websites/website/pages/menus.rst

```diff
@@ -0,0 +1,87 @@
+=====
+Menus
+=====
+
+Menus are used to organize your website’s content effectively and help visitors navigate through
+your web pages effectively. User-friendly and well-structured website menus also play a crucial role
+in improving :doc:`search engine rankings <seo>`.
+
+Odoo allows you to customize the content and appearance of these menus to your needs.
+
+Menu editor
+===========
+
+To edit your website's menu, go to :menuselection:`Website --> Site --> Menu Editor`. From there,
+you can:
+
+- rename a menu item or change its URL using the :guilabel:`Edit Menu Item` icon;
+- delete a menu item using the :guilabel:`Delete Menu Item` icon;
+- move a menu item by drag and dropping it to the desired place in the menu;
+- create a drop-down menu by drag and dropping the sub-menu items to the right, underneath their
+  parent menu.
```

Comment:
Capitals to start the sentences?
I think I would put the actions (rename, delete, move a menu item, create a drop down menu) in bold.

Reply:
I put the actions in bold, but didn't use capital letters as each list item continues the sentence "From there, you can:"